ext/fuzztest.cmd

```diff
@@ -9,8 +9,8 @@
 
 git clone https://github.com/google/fuzztest.git
 cd fuzztest
-: # There is no tagged release as of 2024/01/22. Pick the earliest commit that contains a needed bug fix.
-git checkout 8f82f46e14cc2484ddecf6d55e46243923f9a036
+: # There is no tagged release as of 2024/01/26. Pick the earliest commit that contains a needed bug fix.
```

Comment:
It would be good to say specifically what the needed bug fix is. Or just say something like "Pick a stable snapshot."

Reply:
I put the reason. Then again, this is a moving target. It's the latest stable for us. Let's hope they start versioning.